sources/packages/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -50,13 +56,21 @@ export class StudyBreaksAndWeeksInDTO {
   unfundedStudyPeriodDays: number;
 }
 
+export class StudyBreaksAndWeeksAPIOutDTO {
+  studyBreaks: StudyBreakAPIOutDTO[];
+  fundedStudyPeriodDays: number;
+  totalDays: number;
+  totalFundedWeeks: number;
+  unfundedStudyPeriodDays: number;
+}
+
 export class EducationProgramOfferingAPIInDTO {
   @Allow()
   offeringName: string;
   @Allow()
-  studyStartDate: Date;
+  studyStartDate: string;
   @Allow()
-  studyEndDate: Date;
+  studyEndDate: string;
```

Comment:
why is the date changed to string in all the places?

Reply:
These are date-only values that are supposed to receive only date, not time.
The offering right now has the problem with the dates that goes back one day after saved and rendered back into the UI.
I started changing the API and I will change the Vue to fix it.
I did not start it I am just following what we have been doing for all date-only values (I guess).